### ISSUE TITLE
File delete button not accessible

### ISSUE THREAD

**Issue by adrianbj** [ryancramerdesign/ProcessWire#1001]
At certain screen widths, you can't see the delete button

![screen shot 2015-03-07 at 9 27 03 am](https://cloud.githubusercontent.com/assets/871211/6542418/3aac8cbc-c4ac-11e4-852a-34dbe8e86e60.png)

You can just see the very top of it.

If you change the width of the viewport just slightly, in either has room to fit on that first line, or it wraps to the next line.

**Comment by ryancramerdesign** [comment]
Interesting, I can see that too. It's like it has wrapped to the next line but for whatever reason the icon doesn't cause the container to grow. It's not until the dimensions wrap to the next line that you then see where it went. Not exactly sure what the solution is for that, will have to experiment. Let me know if you have any ideas.

**Comment by somatonic** [comment]
The container <p> has a height of 25px. So it's simple :D

**Comment by adrianbj** [comment]
No longer relevant since the image upgrades in PW 3.x
